.gitpod.yml

```diff
@@ -0,0 +1,17 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Compile the protocol buffers & launch the backend with docker-compose
+    init: |
+      cd app
+      docker run --rm -w /app -v $(pwd):/app couchers/grpc ./generate_protos.sh
+      docker-compose up --build
```

Comment:
Does this command already include a frontend docker container? Maybe it should specify `backend proxy postgres`

Reply:
It doesn't look like we do, I'm guessing it was removed since the prod app are a bunch of minified js and resource files on S3 so we don't actually need it running in a container?